independent-publisher-connectors/WebEx/apiDefinition.swagger.json

```diff
@@ -0,0 +1,291 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Webex Integration",
```

Comment:
@ocarhui I would like to bring your attention to an existing connector for [Webex](https://learn.microsoft.com/en-us/connectors/webex/). Please make sure that you have different actions and triggers than the existing one and it should not have the same name as well.

Reply:
Hi Amjed, the actions and triggers are different to the existing one. This Webex connector is for setting up meetings and creating meeting invitee. Shall I change the name to Webex connector for Webex (Meeting) connector?

Reply:
@ocarhui thank you so much for the confirmation. 

No need to change the name since it is "Webex Integration" and not "Webex" :-). 

Please just change the folder name from "Webex" to "Webex Integration"
![image](https://user-images.githubusercontent.com/97551789/219448030-244a97ae-c351-495d-9b28-91d9c707751c.png)

Please resolve all the above conversations.

Please let me know if you have any questions.

Thank you so much again.